### PR TITLE
Add code in group.py

### PR DESCRIPTION
**Pull request by shashvatshukla**
Answers UCL-MPHY0021-21-22/RSE-Classwork#7

### CODE REVIEW

**Review by erg43 — COMMENTED** [review]
Looks nice, pretty much like the example

**Comment by JC-Hammer97** [comment]
Good use of the relationship type as the key!

**Comment by tomholdendye** [comment]
Very nicely formatted, and we like that jobs is written as a list, to allow for multiple forms of employment!

**Review by zyxxyztp — COMMENTED** [review]
I think John also Nash's cousin.

File: group.py [hunk 1]
```diff
@@ -2,4 +2,33 @@
 
 # Your code to go here...
```
Comment (gittmp) [inline group.py:3]: Simple but effective approach! Love the ability to have multiple connections/jobs

File: group.py [hunk 2]
```diff
@@ -2,4 +2,33 @@
...
+                "jobs": ["Artist"]},
+                "connections": {
+                    "friend": ["Jill"]
+                    }
+            "John": {
+                "age": 27,
+                "jobs": ["Writer"]},
+                "connections": {
+                    "partner": ["Jill"]
+                    }
```
Comment (gittmp) [inline group.py:24]: What would happen if tried to search for John's friends? Or Zalika's partner?

**Pull request closed without merging**